butterknife-annotations/src/main/java/butterknife/BindDrawable.java

```diff
@@ -16,4 +16,10 @@
 public @interface BindDrawable {
   /** Drawable resource ID to which the field will be bound. */
   int value();
+
+  /** Color attribute resource ID that is used to tint the drawable. */
+  int tint() default NONE;
+
+  /** Default value for {@link #tint()}. */
+  int NONE = -1;
```

Comment:
This should be 0, not -1. -1 is technically a valid, possible resource ID. 0 is not a valid resource ID.

Also I'd just inline the default and skip the constant.